pages/docs/index.tsx

```diff
@@ -48,7 +48,14 @@ const Docs = ({ bitbucket_auth_url }: { bitbucket_auth_url: string }) => {
 			heading: "Github",
 			flag: true,
 			content: [
-				{ subHeading: "Sign up with github", article: "Sign in on Vibinex using GitHub" },
+				{
+					subHeading: "Sign up with github",
+					article: <span className="text-blue-500 cursor-pointer" onClick={() => login(
+						getAndSetAnonymousIdFromLocalStorage(),
```

Comment:
this is to help add event on login button, right?

Reply:
Yes.